src/main/java/com/blockout/controls/Image.java

```diff
@@ -97,7 +97,7 @@ private void loadMapDimensions()
             }
             catch (IOException e)
             {
-                Log.logger.error(e);
+                e.printStackTrace();
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) Use a logger to log this exception. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AS1148)